modules/yarn.lock

```diff
@@ -1045,6 +1045,14 @@
     "@clayui/icon" "^3.0.1"
     classnames "^2.2.6"
 
+"@clayui/button@^3.1.0":
```

Comment:
Hey @oliveiraaraujo, as a rule of thumb, **you should never commit changes to `yarn.lock`**. Whenever you see changes here, chances are there's something wrong in your PR.

IN this case, I'd say a different version from the ones already in master are being added. Run `gradlew formatSource -a` to get the the proper versions updated in your `package.json` file and then revert the changes here.

If they persist, then chances are someone forgot to add them, in which case, please send this through someone in the Frontend Infra team (@carloslancha, @wincent, @julien...) so we can analyze what happened and try to correct it for the future.